index.html

```diff
@@ -89,13 +89,15 @@ <h1 class="work-title">Tonic</h1>
     </article>
 
     <article class="portfolio-work">
-       <img class="work-image" src="./assets//Snapshoot Portfolio 4.svg" alt="">
-   <div class="work-description swap-class">
+       <img class="work-image" src="./assets/Snapshoot Portfolio-1.png" alt="">
+   <div class="work-description">
        <h1 class="work-title">Multi-Post <br>Stories</h1>
        <ul class="job-description">
            <li>FACEBOOK</li>
-           <li>FULL STACK DEV</li>
-           <li>2015</li>
+           <ul class="details">
+            <li>FULL STACK DEV</li>
+            <li>2015</li>
+          </ul>
        </ul>
        <p>Experimental content creation feature that allows users to 
         add to an existing story over the course of a day without 
```

Comment:
- [Optional] Please,it is clear that the card size here is fixed try to use more responsive scales  to make it better 
![image](https://user-images.githubusercontent.com/74744787/208268249-f7b0bf8b-8e85-430a-bab3-b34db0ce65f8.png)